webapp/package.json

```diff
@@ -21,7 +21,7 @@
     "@mui/material-nextjs": "^5.16.8",
     "@mui/x-tree-view": "^7.23.0",
     "@octokit/rest": "^20.1.1",
-    "next": "14.2.10",
+    "next": "14.2.15",
```

Comment:
When we bump `next`, we also have to bump `eslint-config-next` in root `package.json`